src/crypto/sha256_sse4.cpp

```diff
@@ -969,27 +969,27 @@ void Transform(uint32_t* s, const unsigned char* chunk, size_t blocks)
 
 /*
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-; Copyright (c) 2012, Intel Corporation 
-; 
-; All rights reserved. 
-; 
+; Copyright (c) 2012, Intel Corporation
```

Comment:
nit: there are a few remaining trailing spaces e.g. in https://github.com/bitcoin/bitcoin/blob/07f0a61ef711a2f75ded3d73545bfabdf2a64fef/src/minisketch/src/fields/clmul_common_impl.h#L39, we could add a PR to https://github.com/bitcoin-core/minisketch to get rid of those as well

Reply:
https://github.com/bitcoin-core/minisketch/pull/98.